### PR TITLE
Made Token limitations more robust

### PR DESCRIPTION
**Pull request by ith-harvey**
# Description of change
## High level
* Altered `README.MD` to reflect a more robust definition of incompatible tokens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,20 @@
 
 The Ajna protocol is a non-custodial, peer-to-peer, permissionless lending, borrowing and trading system that requires no governance or external price feeds to function. The protocol consists of pools: pairings of quote tokens provided by lenders and collateral tokens provided by borrowers. Ajna is capable of accepting fungible tokens as quote tokens and both fungible and non-fungible tokens as collateral tokens.
 
-## Limitations
-- The following types of tokens are incompatible with Ajna, and no countermeasures exist to explicitly prevent creating a pool with such tokens:
-	- Fungible tokens whose balance rebases.
-	- NFTs which charge a fee on transfer.
-	- Fungible tokens with more than 18 decimals or 0 decimals.
+## Accepted tokens:
+- Fungible tokens (following the [ERC20 token standard](https://eips.ethereum.org/EIPS/eip-20)) whose balance rebases.
```
Comment (MikeHathaway) [inline README.md]: We don't accept fungible tokens whose balance rebases

Comment (ith-harvey) [reply README.md]: whoopsie, good catch. Updated -> https://github.com/ajna-finance/contracts/pull/557/commits/3ad2309aece3981e71e7766d3d0e93df9473b47b

**Review by prat-gpt — APPROVED** [review]
LGTM

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**